app/main/controller/audio_transcription_controller.py

```diff
@@ -99,5 +99,8 @@ def get_transcription():
                     transcriptionResponse = requests.get(transcriptionUri)
                     transcriptionResponseDict = json.loads(transcriptionResponse.text)
                     result['transcription'] = transcriptionResponseDict['results']['transcripts'][0]['transcript']
+                if job_status == "FAILED":
```

Comment:
are these the only two job status? (if there was another one we might skip and fail to log?)

Reply:
fair - no idea. I'll add an else.